src/content/chapter3_data_types/lesson04_record_pattern_matching/en.html

```diff
@@ -0,0 +1,12 @@
+<p>
+  It is possible to pattern match on a record, this allows for the extraction 
+  of multiple field values from a record into distinct variables in a concise and efficient way.
+</p>
+<p>
+  the let key word can only match on single variant custom types. For types with more variants 
+  a case statement must be used.
+</p>
+<p>
+  It is possible to use underscore <code>_</code> or the spread syntax <code>..</code> to 
+  discard fields that are not required.
```

Comment:
It would be good to explain the difference between them here, and show the difference in the code example.